pydata_sphinx_theme/static/css/theme.css

```diff
@@ -0,0 +1,76 @@
+/*****************************************************************************
+ * Custom font
+**/
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+:root {
+  /*****************************************************************************
+  * Font size
+  **/
+  --font-size-base: 15px; /* base font size - applied at body / html level */
+
+  /* heading font sizes */
+  --font-size-h1: 36px;
+  --font-size-h2: 32px;
+  --font-size-h3: 26px;
+  --font-size-h4: 21px;
+  --font-size-h5: 18px;
+  --font-size-h6: 16px;
+
+  /* smaller then heading font sizes*/
+  --font-size-milli: 12px;
+
+  /*****************************************************************************
+  * Font family
+  **/
+  --font-family-base: 'Lato', sans-serif;
+  --font-family-heading: 'Open Sans', sans-serif;
+
+  /*****************************************************************************
+  * Color
+  * 
+  * Colors are defined in rgb string way, "red, green, blue"
+  **/
+  --color-primary: 0, 123, 255;
+  --color-success: 40, 167, 69;
+  --color-info: 23, 162, 184;
+  --color-warning: 255, 193, 7;
+  --color-danger: 220, 53, 69;
+  --color-text-base: 51, 51, 51;
+  --color-h1: var(--color-primary);
+  --color-h2: var(--color-primary);
+  --color-h3: var(--color-text-base);
+  --color-h4: var(--color-text-base);
+  --color-h5: var(--color-text-base);
+  --color-h6: var(--color-text-base);
+  --color-paragraph: var(--color-text-base);
+  --color-link: 0, 91, 129;
+  --color-link-hover: 227, 46, 0;
+  --color-sidebar-link: 0, 0, 0;
+  --color-sidebar-link-hover: 19, 6, 84;
+  --color-sidebar-link-active: 19, 6, 84;
+  --color-toc-link: 119, 117, 122;
+  --color-toc-link-hover: 38, 38, 38;
+  --color-toc-link-active: 19, 6, 84;
```

Comment:
* For `sidebar`, the link-hover and link-active have the same color, but for `toc` a different one. Should we for simplicity make `toc-link-hover` the same as well? (unless there is a design-wise reason to make it different?)
* If those 4 have the same color, maybe we should make a variable for that color? (as I think users wanting to override this color, will typically want to override all of them)